buildSrc/src/main/java/uk/gov/hmrc/components/buildsrc/ComposeDependencies.kt

```diff
@@ -21,28 +21,28 @@ class ComposeDependencies {
         const val targetSdk = 34
         const val compileSdk = 34
         const val kotlinCompilerExtensionVersion = "1.4.8"
-        const val jetpackNavigation = "2.7.6"
-        const val lifecycleVersion = "2.6.2"
+        const val jetpackNavigation = "2.7.7"
+        const val lifecycleVersion = "2.7.0"
     }
 
     object Libs {
-        const val compose_bom = "androidx.compose:compose-bom:2024.02.01"
-        const val compose_foundation = "androidx.compose.foundation:foundation"
-        const val compose_material3 = "androidx.compose.material3:material3"
-        const val compose_material = "androidx.compose.material:material:"
-        const val compose_tooling = "androidx.compose.ui:ui-tooling"
-        const val compose_tooling_preview = "androidx.compose.ui:ui-tooling-preview"
-        const val compose_ui = "androidx.compose.ui:ui"
-        const val compose_uiTest = "androidx.compose.ui:ui-test-junit4"
```

Comment:
Previously, we did not add an explicit version as we are using the bom. Is it necessary to add the versions here?